exporter/collector/metricsexporter.go

```diff
@@ -230,10 +316,24 @@ func (m *metricMapper) gaugePointToTimeSeries(
 	}
 }
 
+// Returns any configured prefix to add to unknown metric name.
+func (m *metricMapper) getMetricNamePrefix(name string) *string {
```

Comment:
Can you add unit test for this with some known_domains?

Reply:
done.